### ISSUE TITLE
Glem no longer works with recent Frog/mbt

### ISSUE THREAD

**Issue by proycon** [GreekPerspective/glem#8]
I'm working on containerisation of glem according to CLARIAH requirements, as LaMachine is deprecated and its current deployment in Nijmegen will cease eventually. With the newer Frog/mbt versions, I seem to run into a problem:

```
[CLAM Dispatcher] Adding to PYTHONPATH: /usr/lib/python3.10/site-packages/webglem
[CLAM Dispatcher] Started CLAM Dispatcher v3.1.6 with webglem.webglem (2022-11-03 16:42:29)
Undefined environment variable: CLAM_DEBUG
Undefined environment variable: CLAM_OAUTH_CLIENT_URL
Undefined environment variable: CLAM_OAUTH_REVOKE_URL
Undefined environment variable: CLAM_OAUTH_SCOPE
[CLAM Dispatcher] Running python3 "/usr/lib/python3.10/site-packages/webglem/webglem_wrapper.py" "/data/glem/webglem-userdata/projects/anonymous/test/clam.xml" "/data/glem/webglem-userdata/projects/anonymous/test/.status" "/data/glem/webglem-userdata/projects/anonymous/test/output/"
[CLAM Dispatcher] Running with pid 23 (2022-11-03 16:42:29)
INITIALISE FROG
20221103:164230:259:requested configuration: /usr/lib/python3.10/site-packages/glem/pretrained_models/herodotus/frog.cfg.template
20221103:164230:259:config read from: /usr/lib/python3.10/site-packages/glem/pretrained_models/herodotus/frog.cfg.template
20221103:164230:259: Parser disabled, because MWU is deselected
20221103:164230:260:mblem-:Initiating lemmatizer...
/usr/lib/python3.10/site-packages/glem/pretrained_models/herodotus/froggen.settings has a PROBLEM!
No DATA_VERSION setting is found but this version of MbT expects at least 2
Cannot read settingsfile /usr/lib/python3.10/site-packages/glem/pretrained_models/herodotus/froggen.settings
ucto: textcat configured from: /usr/local/share/ucto/textcat.cfg
20221103:164231:255:Initialization failed for: [tagger] 
terminate called after throwing an instance of 'std::runtime_error'
  what():  Frog init failed
Traceback (most recent call last):
  File "/usr/lib/python3.10/shutil.py", line 815, in move
    os.rename(src, real_dst)
FileNotFoundError: [Errno 2] No such file or directory: 'input/input-11f07321e87fbc0b.txt.lastrun.wlt.txt' -> '/data/glem/webglem-userdata/projects/anonymous/test/output/input-11f07321e87fbc0b.out.txt'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.10/site-packages/webglem/webglem_wrapper.py", line 107, in <module>
    move(inputfilepath+'.lastrun.wlt.txt',outputdir+basename + '.out.txt')
  File "/usr/lib/python3.10/shutil.py", line 835, in move
    copy_function(src, real_dst)
  File "/usr/lib/python3.10/shutil.py", line 434, in copy2
    copyfile(src, dst, follow_symlinks=follow_symlinks)
  File "/usr/lib/python3.10/shutil.py", line 254, in copyfile
    with open(src, 'rb') as fsrc:
FileNotFoundError: [Errno 2] No such file or directory: 'input/input-11f07321e87fbc0b.txt.lastrun.wlt.txt'
[CLAM Dispatcher] Process ended (2022-11-03 16:42:32, 2.504294s) 
[CLAM Dispatcher] Updating project index
[CLAM Dispatcher] Removing temporary files
[CLAM Dispatcher] Status code out of range (256), setting to 127
[CLAM Dispatcher] Finished (2022-11-03 16:42:32), exit code 127, dispatcher wait time 2.5s, duration 2.504917s
```

**Comment by kosloot** [comment]
> /usr/lib/python3.10/site-packages/glem/pretrained_models/herodotus/froggen.settings has a PROBLEM!
No DATA_VERSION setting is found but this version of MbT expects at least 2

Recent Mbt and Frog (using timbl) versions are better Utf8 aware. This required retraining of a lot of the Frog data-files.
Hence the version=2 requirement.
So you should 'simply' retrain the data-files for Mbt etc.

As a quick and unreliable hack, you could add the line
>  DATA_VERSION 2

at the top of the settingsfile

**Comment by proycon** [comment]
Thanks to your pull request, this is solved now. I released v1.3.1 with the necessary changes.
